proto/spire/api/server/entry/v1/entry.proto

```diff
@@ -176,3 +191,24 @@ message GetAuthorizedEntriesResponse {
     // The authorized entries.
     repeated spire.api.types.Entry entries = 1;
 }
+
+message SyncAuthorizedEntriesRequest {
+    // An output mask indicating which fields are set in the response.
+    spire.api.types.EntryMask output_mask = 1;
+
+    // IDs of the entries to fetch in full. Sent by the client in response to
+    // a sparse entry.
+    repeated string ids = 2;
+}
+
+message SyncAuthorizedEntriesResponse {
+    // Full or sparse entries, depending on the sparse field. See
+    // SyncAuthorizedEntries for details.
+    repeated spire.api.types.Entry entries = 1;
```

Comment:
I'm ok with this, but, may we use oneof? and return another type of "entries"? instead of depending of an sparse?